faq/css/faq.css

```diff
@@ -1,1377 +1,440 @@
-html {
-  color: #222;
-  font-size: 1em;
-  line-height: 1.4;
-}
-::-moz-selection {
-  background: #b3d4fc;
-  text-shadow: none; }
-
-::selection {
-  background: #b3d4fc;
-  text-shadow: none; }
-hr {
-  display: block;
-  height: 1px;
-  border: 0;
-  border-top: 1px solid #ccc;
-  margin: 1em 0;
-  padding: 0; }
-audio,
-canvas,
-iframe,
-img,
-svg,
-video {
-  vertical-align: middle; }
-fieldset {
-  border: 0;
-  margin: 0;
-  padding: 0; }
-textarea {
-  resize: vertical; }
-h1, h2 {
-  text-transform: uppercase; }
+html{color:#222;font-size:1em;line-height:1.4}
```

Comment:
CSS should not be minimized in repo. Should be minimized by gulp for production builds